airflow/www/templates/airflow/cluster_activity.html

```diff
@@ -43,6 +43,7 @@
   <script>
     const stateColors = {{ state_color_mapping|tojson }};
     const autoRefreshInterval = {{ auto_refresh_interval }};
+    const standaloneDagProcessor = {{ standalone_dag_processor }} ;
```

Comment:
@bbovenzi if there is a way to make this a boolean value then it would be much better. Currently when I pass in a boolean from `views.py` it throws an error because Python's `True` is capitalized.

Reply:
const standaloneDagProcessor = {{ standalone_dag_processor }} === 'True';
This will convert it to boolean